worldengine/cli/main.py

```diff
@@ -9,13 +9,14 @@
     draw_precipitation_on_file, draw_grayscale_heightmap_on_file, draw_simple_elevation_on_file, \
     draw_temperature_levels_on_file, draw_riversmap_on_file
 from worldengine.plates import world_gen, generate_plates_simulation
+from worldengine.imex import export
```

Comment:
imex is used other higher profile projects, I've not seen any that have used importexport and I'm not particularly fond of that either. ;)